packages/svelte-check/README.md

```diff
@@ -108,7 +108,7 @@ The output concludes with a `COMPLETED` message that summarizes total numbers of
 ###### Example:
 
 ```
-1590680326807 COMPLETED 20 FILES 21 ERRORS 1 WARNINGS
+1590680326807 COMPLETED 20 FILES 21 ERRORS 1 WARNINGS 3 FILES_WITH_PROBLEMS
```

Comment:
I included underscores so the parts remain separated by whitespace. There may be a one-word name to prefer, I chose this to be clearer.

I didn't change the documentation text above, the changes I tried seemed to make it more complicated without adding anything useful, maybe there's a small tweak that could be made:

> The output concludes with a `COMPLETED` message that summarizes total numbers of files, errors and warnings that were encountered during the check.